coverage-reporter/build.gradle.kts

```diff
@@ -0,0 +1,61 @@
+// SPDX-License-Identifier: MIT
+// SPDX-FileCopyrightText: 2023
+
+plugins {
+    kotlin("jvm") version "1.9.21"
+    `maven-publish`
+    idea
+    `java-gradle-plugin`
+    id("signing")
+}
+
+group = "io.github.amosproj"
```

Comment:
can we align this with the override-plugin?
so as id something like
`io.github.amos-pitmutationmate.pitmutationmate.coverage-reporter`

Reply:
Yes I can change that. I am just struggling with uploading the plugin at all. After I did that I will try to change it.